smrf/data/loadGrid.py

```diff
@@ -297,7 +300,7 @@ def load_from_netcdf(self):
                 except:
                     pass
                 df = df[self.start_date:self.end_date]
-                setattr(self, v, df.tz_localize(tz=self.time_zone))
+                setattr(self, v, df.tz_convert(tz=self.time_zone))
```

Comment:
You might want to try flipping these two lines. `start_date` and `end_date` are tz aware but the data frame index doesn't appear to be at this point. This is where I would debug, seeing which one of these dates is not tz aware.